### DISCUSSION TITLE
What is the difference between release and full-release?

### DISCUSSION THREAD

**Discussion by wimdeblauwe** [jreleaser/jreleaser#1025] [Q&A]
The Maven plugin has [jrelease:release](https://jreleaser.org/guide/latest/tools/jreleaser-maven.html#_jreleaserrelease) and [jrelease:full-release](https://jreleaser.org/guide/latest/tools/jreleaser-maven.html#_jreleaserfull_release) but it is unclear from those documentation pages what the difference between those is.

**Comment by aalmiray** [comment]
`release` is responsible for creating a Git release https://jreleaser.org/guide/latest/workflow.html#_release
`full-release` performs a release, publishes packagers, and announces the release https://jreleaser.org/guide/latest/workflow.html#_full_release
